artcommon/artcommonlib/konflux/konflux_db.py

```diff
@@ -152,11 +152,54 @@ async def search_builds_by_fields(self, start_search: datetime, end_search: date
         self.logger.debug('Found %s builds', results.total_rows)
         return results
 
+    async def search_builds_by_fields_with_window(
+            self,
+            start_search: typing.Optional[datetime] = None,
+            end_search: typing.Optional[datetime] = None,
+            search_window: int = 90,
+            where: typing.Optional[typing.Dict[str, typing.Any]] = None,
+            extra_patterns: typing.Optional[typing.Dict] = None,
+            order_by: str = '',
+            sorting: str = 'DESC',
+            limit: typing.Optional[int] = None) -> typing.AsyncGenerator[Row, None]:
+        """ Execute search_builds_by_fields() with a window of given days.
+
+        :param start_search: The start time of the search (inclusive). If None, the search starts 3 years ago.
+        :param end_search: The end time of the search (exclusive). If None, the search ends now.
+        :param search_window: The number of days in the search window. Default is 90 days.
```

Comment:
`get_latest_build()` defines a window size of 30 days. I think we should define a unique window size and store it as a constant at class or module level.

Reply:
I added 2 constants for both of the functions.